src/index.html

```diff
@@ -7,7 +7,49 @@
     <title>Stars</title>
     <link rel="stylesheet" href="./style.css">
   </head>
+
   <body>
-    <h1>Stars</h1>
+    <div class="stars--0 stars">
+      <div class="stars__star"></div>
+      <div class="stars__star"></div>
+      <div class="stars__star"></div>
+      <div class="stars__star"></div>
+      <div class="stars__star"></div>
+    </div>
+    <div class="stars--1 stars">
```

Comment:
```suggestion
    <div class="stars--0 stars">
      <div class="stars__star"></div>
      <div class="stars__star"></div>
      <div class="stars__star"></div>
      <div class="stars__star"></div>
      <div class="stars__star"></div>
    </div>
    
    <div class="stars--1 stars">
```

Add empty line between multiple sibling blocks of html, fix all such cases